src/constants.ts

```diff
@@ -11,8 +11,8 @@ export const WSTETH = ethers.utils.formatBytes32String('04').slice(0, 14)
 export const STETH = ethers.utils.formatBytes32String('05').slice(0, 14)
 export const LINK = ethers.utils.formatBytes32String('06').slice(0, 14)
 export const ENS = ethers.utils.formatBytes32String('07').slice(0, 14)
-export const YVDAI = ethers.utils.formatBytes32String('08').slice(0, 14)
-export const YVUSDC = ethers.utils.formatBytes32String('09').slice(0, 14)
+export const YVDAI = ethers.utils.formatBytes32String('yvDAI').slice(0, 14)
```

Comment:
We are following the sequential ordering of asset ids, with the asset id determined at the time that the proposal is submitted, please choose '08' and '09' here.

Those ids might change by the time we get to mainnet, but that change would be reflected only in the environments repo. The source of truth for asset ids will always be the Cauldron, and you can recover all asset ids either by filtering the events, or querying the Cauldron for sequentially increasing asset ids until you get one that gives you the zero address.